editor/editor_resource_picker.cpp

```diff
@@ -1120,6 +1123,12 @@ EditorResourcePicker::EditorResourcePicker(bool p_hide_assign_button_controls) {
 		assign_button->add_child(preview_rect);
 	}
 
+	quick_load_button = memnew(Button);
+	quick_load_button->set_flat(false);
```

Comment:
```suggestion
```
This is default.
(I know other buttons set it, but it's unnecessary)

Reply:
Done!